metadata.json

```diff
@@ -12,8 +12,10 @@
     "bundle",
     "security",
     "compliance",
-    "NIST 800-53",
-    "DISA STIG"
+    "NIST",
+    "800-53",
+    "DISA",
```

Comment:
These really shouldn't be split.

If spaces aren't allowed, use underscores.

Reply:
Why would anyone search for those? Splitting them up means they would show up in searches for any one of the tags, right?

Reply:
Hrm...yeah, I guess you're right. Just much less precise than would be nice. That said, OK!